File: JavaScript/watermelon.js
function splitWatermelon(weight) {
  //% Gives the remainder of the operation. So in this case any even number % 2 will be 0.
  if (weight % 2 == 0) {
    console.log("YES");
    // If the weight cannot be split evenly in half, then the number is odd and will never be able to be split into two even parts.
  } else {
    console.log("NO");
  }
}

File: C++/Watermelon.cpp
#include<bits/stdc++.h>
using namespace std;
 
int main()
{
       int n;
       cin>>n;
       if(n%2==0 && n>2)    //If the number is divided in 2 parts then it should be even
        cout<<"YES";
       else
        cout<<"NO";
       return 0;
}